react/Breadcrumbs/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import styles from './styles.styl'
+import { coolGrey } from '../palette'
+import Icon from '../Icon'
+
+const BreadcrumbSeparator = () => (
+  <span className={styles.BreadcrumbSeparator}>/</span>
+)
+
+const BreadcrumbItem = ({
+  name,
+  onClick,
+  isCurrent = false,
+  tag = 'span',
+  showSeparator = false
+}) => {
+  const Tag = tag
+  return (
+    <div
+      className={cx(styles.BreadcrumbItem, {
+        [styles['BreadcrumbItem--current']]: isCurrent
+      })}
+    >
+      <Tag
+        onClick={onClick}
+        className={cx({
+          'u-clickable': onClick
+        })}
+      >
+        {name}
+      </Tag>
+      {showSeparator && <BreadcrumbSeparator />}
+    </div>
+  )
+}
+
+const itemPropTypes = {
+  name: PropTypes.string.isRequired,
+  onClick: PropTypes.func,
+  tag: PropTypes.element
+}
+
+BreadcrumbItem.propTypes = {
+  ...itemPropTypes,
+  isCurrent: PropTypes.bool,
+  showSeparator: PropTypes.bool
+}
+
+const Breadcrumbs = ({ items, className, color }) => {
+  const previousItems = items.slice(0, -1)
+  const [lastPreviousItem] = previousItems.slice(-1)
+  const [currentItem] = items.slice(-1)
+
+  return (
+    <div className={cx(styles.Breadcrumb, className)}>
+      {items.length > 1 && (
+        <Icon
+          icon={'left'}
+          color={
+            color === 'primary' ? 'var(--primaryContrastTextColor)' : coolGrey
+          }
+          className={cx(styles.Breadcrumb__previousButton, styles[color])}
+          onClick={lastPreviousItem.onClick}
+        />
+      )}
+      <div className={cx(styles.Breadcrumb__items, styles[color])}>
+        <div className={styles.Breadcrumb__previousItems}>
+          {previousItems.map(({ name, onClick, tag }, index) => (
+            <BreadcrumbItem
+              key={name}
```

Comment:
```suggestion
              key={index}
```

I think `name` might not be unique enough, for example in a folder architecture like `Jobs/Cozy/Cozy/Whatever` we would have identical keys and i'm not sure which one react would keep if only one is removed.